### ISSUE TITLE
'adcStart' was not declared in this scope

### ISSUE THREAD

**Issue by chinswain** [e-tinkers/esp32-adc-calibrate#1]
Hi,
I get the below errors when running the code in Arduino 1.8.13 with the latest arduino-esp32 build:

 ```
error: 'adcStart' was not declared in this scope
error: 'adcEnd' was not declared in this scope
```

**Comment by e-tinkers** [comment]
What is GPIO pin value you pass in to the `adcStart()` function? GPIO 35 that I used on a generic [ESP32 Dev Baord](https://github.com/espressif/arduino-esp32#esp32dev-board-pinmap), is equivalent to A7 in Arduino. If you pass in an invalid pin (e.g. A1 or A2), you will get the error as those are not defined in ESP32 as valid ADC inputs.

**Comment by chinswain** [comment]
I've tried a few boards including a generic one, I've checked the schematics on a few to confirm the pins are correct. The error occurs when I compile against "ESP Dev Module" so the pin mapping should be 25/35 as your example.

![image](https://user-images.githubusercontent.com/4403897/87768420-ffd05a80-c813-11ea-9166-cacfc6dd2e16.png)

I've posted a message in the arduino-esp32 chat, I'll update here if there's any news.
